pkg/broadcast/broadcast.go

```diff
@@ -0,0 +1,81 @@
+package broadcast
```

Comment:
A little bit of godocs coments would be really helpful on this file and a little at the package level too.